alembic/versions/2023_12_15_27ec5c4c0380_add_concentration_for_cfdna.py

```diff
@@ -0,0 +1,38 @@
+"""Add concentration for cfdna
+
+Revision ID: 27ec5c4c0380
+Revises: b105b426af99
+Create Date: 2023-12-15 16:47:19.031328
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "27ec5c4c0380"
+down_revision = "b105b426af99"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        table_name="application",
+        column=sa.Column(
+            "sample_concentration_minimum_cfdna",
```

Comment:
Do we really want to have columns for each possible type of dna? Seems more intuitive to have a separate table with the limitations which use an enum for the different types of dna, like cfdna. I don't like having columns with such specific names, seems like it should be an enum/column in a separate table

Reply:
seems a little messy and if we want to add new dna types we would have to add new columns

Reply:
@seallard No one was super happy with any solution but in the end it was decided that this one was the preferable. I agree that your solution is probably the cleanest if this was intended for all applications, but currently sample concentrations are manually handled and updated, meaning that ease of use was preferred. We discussed whether we wanted to add another table or two, but it seemed a bit overkill to reroute all concentrations and their handling.

Reply:
If we in the future would like to expand this, which there are no current plans of doing, I agree that we would want to refactor.

Reply:
Alright! I guess we can refactor it later if we run into it. It is probably possible to hide away the database internals in the admin interface, so I don't think it would necessarily be more difficult to use. Lets leave it for now then :+1: